.travis.yml

```diff
@@ -0,0 +1,32 @@
+language: elixir
+matrix:
+  include:
+    - elixir: 1.4
+      otp_release: 18.3
+    - elixir: 1.4
+      otp_release: 19.3
+    - elixir: 1.4
+      otp_release: 20.3
+    - elixir: 1.5
+      otp_release: 18.3
+    - elixir: 1.5
+      otp_release: 19.3
+    - elixir: 1.5
+      otp_release: 20.3
+    - elixir: 1.6
+      otp_release: 19.3
+    - elixir: 1.6
+      otp_release: 20.3
+    - elixir: 1.6
+      otp_release: 21.0
+    - elixir: 1.7
+      otp_release: 19.3
+    - elixir: 1.7
+      otp_release: 20.3
+    - elixir: 1.7
+      otp_release: 21.0
```

Comment:
I personally tend to be a bit more environment friendly (:D) and only the mins and max, so 1.4 and 1.7. Thoughts?

Reply:
Oh, I just copied the file from the poller repo, we run telemetry and poller builds on those versions.

Reply:
Ship it then!